ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.Data/Repositories/ProjectRepository.cs

```diff
@@ -1,14 +1,41 @@
 ﻿namespace ExadelTimeTrackingSystem.Data.Repositories
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
     using ExadelTimeTrackingSystem.Data.Configuration.Abstract;
     using ExadelTimeTrackingSystem.Data.Models;
     using ExadelTimeTrackingSystem.Data.Repositories.Abstract;
+    using MongoDB.Driver;
 
     public class ProjectRepository : MongoRepository<Project>, IProjectRepository
     {
+        private readonly IMongoDatabase _database;
+
         public ProjectRepository(IMongoDbSettings settings)
             : base(settings)
         {
+            var client = new MongoClient(settings.ConnectionString);
```

Comment:
it's already performed in base constructor (you call it with `base(settings)`) of MongoRepository, so same job is done twice, mongo connection established twice, you don't need it